src/tools/pub/package-layout.md

```diff
@@ -299,18 +302,17 @@ enchilada/
     style.css
 {% endprettify %}
 
-For web packages, place HTML, CSS, JavaScript, and Dart code under
-the `web` directory. Any Dart web entrypoints (in other words,
-Dart scripts that are referred to in a `<script>` tag) go under `web` and
-not `lib`. This ensures that `package:` imports can be resolved correctly.
+For web packages, place entrypoint code&mdash;Dart scripts that include
+`main()` and supporting files, such as CSS or HTML&mdash;under
+`web`.  This ensures that `package:` imports can be resolved correctly.
```

Comment:
I still don't know what this sentence ("This ensures...") really means. (package imports from who? to what?) Delete this sentence?